nuxt.config.ts

```diff
@@ -86,6 +86,20 @@ export default defineNuxtConfig({
     build: {
       target: 'esnext',
     },
+    optimizeDeps: {
+      include: [
+        '@tiptap/vue-3', 'string-length', 'vue-virtual-scroller', 'emoji-mart', 'iso-639-1',
+        '@tiptap/extension-placeholder', '@tiptap/extension-document', '@tiptap/extension-paragraph',
+        '@tiptap/extension-text', '@tiptap/extension-mention', '@tiptap/extension-hard-break',
+        '@tiptap/extension-bold', '@tiptap/extension-italic', '@tiptap/extension-code',
+        '@tiptap/extension-history', 'prosemirror-state', 'browser-fs-access', 'blurhash',
+        '@vueuse/integrations/useFocusTrap', '@tiptap/extension-code-block', 'prosemirror-highlight',
+        '@tiptap/core', 'tippy.js', 'prosemirror-highlight/shiki', '@fnando/sparkline',
+        '@vueuse/gesture', 'github-reserved-names', 'file-saver', 'slimeform', 'vue-advanced-cropper',
+        'workbox-window', 'workbox-precaching', 'workbox-routing', 'workbox-cacheable-response',
+        'workbox-strategies', 'workbox-expiration',
+      ],
+    },
```

Comment:
We could probably remove the ones that would be detected by the scanner, but I think it is ok for now to include all of these 👍🏼

Reply:
Some dependencies only detected when matching the case: for example, github-reserved-names and file-saver when visiting the settings > users and about pages

When visiting  a new page and found a new entry in the console I just add them to the list.

![imagen](https://github.com/elk-zone/elk/assets/6311119/241df17e-ab8e-4378-9d31-b9957391c218)

Reply:
perfect 👍🏼